packages/mobile/components/buttons/ProfileButton.svelte

```diff
@@ -0,0 +1,97 @@
+<script lang="typescript">
+    import { appSettings } from '@core/app'
+    import { Icon, NetworkIcon, Text } from 'shared/components'
+    import { FontWeight, TextType } from 'shared/components/Text.svelte'
+    import { getInitials as _getInitials } from 'shared/lib/helpers'
+    import { NetworkProtocol, NetworkType } from '@core/network'
+
+    export let name = ''
+    export let id = ''
+    export let networkType: NetworkType
+    export let networkProtocol: NetworkProtocol
+    export let disabled = false
+    export let hidden = false
```

Comment:
What do we need this for?

Reply:
Do you just mean hidden or should we remove disabled as well? I guess there is no reason to show a disabled profile button?

Reply:
`hidden` I guess it comes from the onboarding buttons that can be hidden based on the enabled/disabled feature flags, so i dont think we will need it here. `disable` i think is useful in case we need it we some reason, is always good being able to disable a button